package.json

```diff
@@ -7,17 +7,21 @@
     "node": ">= 10.0.0 <11.0.0"
   },
   "scripts": {
+    "ci-checks": "npm run lint && npm run test-with-coverage",
```

Comment:
Yeah, I think it’s ok to run 2 separate scripts in our CI/CD.
But for "test-with-coverage" I'd still have a separate npm script so that I can check coverage with 1 click in my IDE instead of writing the command in the console every time :)